doc/manual/source/protocols/json/hash.md

```diff
@@ -0,0 +1,7 @@
+{{#include hash-v1-fixed.md}}
+
+<!--
+## Raw Schema
+
+[JSON Schema for Hash v1](schema/hash-v1.json)
+-->
```

Comment:
It would be nice to generate the raw JSON from the YAML, but I didn't do that yet. I think it can wait.

(We could also go back to JSON from YAML for the source files, but I think that is a bit bad for git diffs and multi-line strings.